### PR TITLE
Improve validation error display

### PR DESCRIPTION
**Pull request by MillerApps**
Should fix issue #17 

TODO:

- [x] Tweak Visuals

### CODE REVIEW

**Comment by vikmeup** [comment]
@MillerApps how does it look in semi bold?

**Comment by MillerApps** [comment]
Maybe a tad too big. What size do you think would be best? simulator screen shot - iphone x - 2018-05-06 at 02 35 35

**Comment by vikmeup** [comment]
@MillerApps learn and explore, this looks awful atm.

**Comment by MillerApps** [comment]
Here is how it looks.

File: Trust/Style/Colors.swift [hunk 1]
```diff
@@ -5,7 +5,8 @@ import UIKit
 struct Colors {
     static let darkBlue = UIColor(hex: "3375BB")
     static let blue = UIColor(hex: "2e91db")
-    static let red = UIColor(hex: "f7506c")
+    static let radicalRed = UIColor(hex: "f7506c")
+    static let red = UIColor(hex: "E32146")
```
Comment (vikmeup) [inline Trust/Style/Colors.swift]: Is this change? can you just add new color name

File: Trust/UI/FormAppearance.swift [hunk 2]
```diff
@@ -44,18 +44,20 @@ struct AppFormAppearance {
...
         if !row.isValid {
             for (index, validationMsg) in row.validationErrors.map({ $0.msg }).enumerated() {
                 let labelRow = LabelRow {
                     $0.title = validationMsg
-                    $0.cell.height = { 20 }
-                    $0.cell.textLabel?.textColor = .red
+                    $0.cell.height = { 40 }
+                }.cellUpdate { cell, _ in
+                        cell.textLabel?.font = AppStyle.error.font
+                        cell.textLabel?.textColor = AppStyle.error.textColor
```
Comment (vikmeup) [inline Trust/UI/FormAppearance.swift]: spacing

File: Trust/Style/Colors.swift (hunk 1 above)
Comment (MillerApps) [reply Trust/Style/Colors.swift]: Not sure what to call it, any suggestions?

File: Trust/UI/FormAppearance.swift (hunk 2 above)
Comment (MillerApps) [reply Trust/UI/FormAppearance.swift]: Done

File: Trust/Style/Colors.swift (hunk 1 above)
Comment (MillerApps) [reply Trust/Style/Colors.swift]: Reverted `static let radicalRed = UIColor(hex: "f7506c")` `back to static let red = UIColor(hex: "f7506c")`

Renamed `static let red = UIColor(hex: "E32146")` to `static let errorRed = UIColor(hex: "E32146")`

**Pull request merged**